app/code/core/Mage/Eav/Model/Entity/Abstract.php

```diff
@@ -982,7 +996,51 @@ protected function _loadModelAttributes($object)
 
         Varien_Profiler::start('__EAV_LOAD_MODEL_ATTRIBUTES__');
 
-        $selects = array();
+        $tableAttributes = array();
+        $attributeTypes  = array();
+
+        foreach ($this->getAttributesById() as $attributeId => $attribute){
+            /** @var Mage_Catalog_Model_Resource_Eav_Attribute $attribute */
+            if ($attribute instanceof  Mage_Catalog_Model_Resource_Eav_Attribute && !$attribute->isStatic()) {
+                $tableAttributes[$attribute->getBackendTable()][] = $attributeId;
+                if (!isset($attributeTypes[$attribute->getBackendTable()])) {
+                    $attributeTypes[$attribute->getBackendTable()] = $attribute->getBackendType();
+                }
+            }
+        }
+
+        // get eav select skeleton
+        $select = $this->_getLoadAttributesSelectSkeleton($object);
+        foreach ($tableAttributes as $table=>$attributes) {
+            try {
+                /** @var Varien_Db_Select $select */
+                $select->reset(Varien_Db_Select::FROM);
+                $select->from(array('eav_value' => $table), null);
+                $values = $this->getReadConnection()->fetchAll($select);
+                uasort($values, function ($value1, $value2){
+                    if ((int)$value1['store_id'] == $value2['store_id']){
+                        return 0;
+                    }
+                    return $value1['store_id'] < $value2['store_id'] ? -1 : 1;
+
+                });
+            } catch (Exception $e) {
+                Mage::printException($e, $select);
+                $this->printLogQuery(true, true, $select);
```

Comment:
I don't think this is the correct way to handle exceptions at this point, perhaps this was debug code left in accidentally.